staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css

```diff
@@ -0,0 +1,88 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.logo-container img {
+   
+    height: 216px;
+    width: 300px;
+}
+
+.image-design-container img {
+   
+    width: 216px;
+    height: 201px;
+    position: absolute;
+    top: 250px;
+}
+
+.menu-container {
+    color: rgb(14, 13, 13);
+
+    position: absolute;
+    top: 150px;
+    left: 310px;
+}
+
+.menu {
+    width: 900px;
+}
+
+.menu li {
+    display: inline;
+}
+
+.menu li a {
+    font-size: 35px;
+}
+
+.menu ul li:not(:last-child)::after {
+    content: "";
+    border: 1px solid black;
+    border-width: 5px 5px 0 0;
+    position: absolute;
+    top: 0;
+    height: 100%;
+    margin-bottom: 10px;
+}
+
+.menu span {
+    margin: 2em
+}
+
+.nameofthevent-container {
+  
+    position: absolute;
+    top: 20px;
+    left: 310px;
+}
+
+.nameofthevent-container p {
+    font-size: 70px
+}
+
+.actual_page .about_title {
+    visibility: collapse;
+}
+
+.actual_page .contact_title {

```

Comment:
Same code as actual_page .about_title, why two classes?

Reply:
I corrected I mean .actual-page.contact-title to be more specific .